src/features/currentTodo.ts

```diff
@@ -1,10 +1,21 @@
-import { createSlice } from '@reduxjs/toolkit';
+import { createSelector, createSlice } from '@reduxjs/toolkit';
 import { Todo } from '../types/Todo';
+import { RootState } from '../app/store';
 
 const initialState = null as Todo | null;
 
 export const currentTodoSlice = createSlice({
   name: 'currentTodo',
   initialState,
-  reducers: {},
+  reducers: {
+    set: (_, { payload }) => payload,
+  },
 });
+
+const currentTodo = (state: RootState) => state.currentTodo;
+
+export const currentTodoSelector = createSelector([currentTodo], value => {
+  return value;
+});
```

Comment:
The selector function can be simplified by directly returning the result of the `currentTodo` function without wrapping it in a function. This will make the code cleaner.